### DISCUSSION TITLE
ODR or NDR ? 🤔

### DISCUSSION THREAD

**Discussion by silverlays** [86Box/86Box#4380] [General]
Hi there 👋

Being an **i486/DX2** and **Pentium** user and exclusively on **MS-DOS**, **Windows 3.1** and **Window 9x**, I was wondering which is the most reliable recompiler to use in 2024 between **ODR** and **NDR**?

ps: I frequently update 86Box to the latest build

**Comment by ghost** [comment]
ODR is more stable for me, but if you use a pentium and 486 it does not matter
and also mention hardware performance. Sometimes NDR is slower.
